code/ui/blocks/src/blocks/ArgTypes.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ArgTypes } from './ArgTypes';
+import * as ButtonStories from '../examples/Button.stories';
+
+const meta: Meta<typeof ArgTypes> = {
+  title: 'Blocks/ArgTypes',
```

Comment:
It's unfortunate that this shows up as "Arg Types" in the sidebar, is there even anything to do about that?

Reply:
Shouldn't this have fixed that 🤔

Reply:
Yeah I thought so too, but it's probably being split by this. https://github.com/storybookjs/storybook/blob/next/code/ui/.storybook/manager.ts#L6

I guess we'd have to special case it, which seems annoying.

Reply:
Oh, why did we do that I wonder?

Reply:
probably to make the stories look good with autotitling.